geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ListIndexManagementDUnitTest.java

```diff
@@ -70,7 +76,8 @@ public static void beforeclass() throws Exception {
         "create index --name=index1 --type=key --expression=id --region=/region1")
         .statusIsSuccess();
     gfsh.executeAndAssertThat(
-        "create index --name=index2 --type=key --expression=key --region=/region1")
+        "create index --name=" + INDEX_WITH_DOT_IN_NAME
```

Comment:
I found an easier way to test this, basically we only need to intercept what's been received by the controller, we don't really need to start a real locator/server for that.

We can piggybag on `JsonSerializationTest`, add another test method there:


@Test
  public void nameWithDot() throws Exception {
    String url = "/v1/regions/region.name/indexes/index.name";
    context.perform(get(url)).andExpect(status().isOk());
    ArgumentCaptor<Index> argumentCaptor = ArgumentCaptor.forClass(Index.class);
    verify(cms).get(argumentCaptor.capture());
    Index index = argumentCaptor.getValue();
    assertThat(index.getRegionName()).isEqualTo("region.name");
    assertThat(index.getId()).isEqualTo("index.name");
  }

Reply:
Yes, I would much prefer to test this closer to the controller!